client/context.go

```diff
@@ -220,9 +220,9 @@ func (ctx Context) PrintOutput(toPrint proto.Message) error {
 	return ctx.printOutput(out)
 }
 
-// PrintOutputLegacy is a variant of PrintOutput that doesn't require a proto type
+// PrintObjectLegacy is a variant of PrintObject that doesn't require a proto type
 // and uses amino JSON encoding. It will be removed in the near future!
-func (ctx Context) PrintOutputLegacy(toPrint interface{}) error {
+func (ctx Context) PrintObjectLegacy(toPrint interface{}) error {
```

Comment:
If you rename `PrintObject` to `PrintProtoMessage` this breaking change could be avoided.

Reply:
WDYM?

Reply:
I meant that if you do as you proposed here https://github.com/cosmos/cosmos-sdk/pull/8107#discussion_r538289342 then this renaming becomes unnecessary

Reply:
The main motivation of this PR is to not use `Output`.